Sources/AsyncConnection.swift

```diff
@@ -3,7 +3,7 @@ public protocol AsyncConnection: AsyncStream {
 }
 
 extension AsyncConnection {
-    public func open(result: (Void throws -> AsyncConnection) -> Void) throws {
+    public func open(_ result: (Void throws -> AsyncConnection) -> Void) throws {
```

Comment:
@Danappelxx I’m okay with typealiases, but not in the library of standards. `Completion<AsyncConnection>` may be clear once you get it, but at first glance you will feel like «wait, what’s the `Completion`? Is that a `struct` or something?» And that doesn’t feels right for the library that intended to be used really really often.

Mainly, my only argument is that «Swift standard library wouldn’t have anything like that». I don’t know if that’s enough, but that’s my point of view 😄